backend/config.py

```diff
@@ -9,6 +9,7 @@
 
 from pathlib import Path
 import json
+from fastapi_mail import ConnectionConfig
```

Comment:
Native implementation will be preferred here.

Reply:
Ok, I can use `aiosmtplib` instead, the same used by `fastapi_mail`. I installed the `fastapi_mail` library because we would end up with a simpler but similar implementation that we would need to maintain.

We can use the `smtplib` module in the standard library, or install `aiosmtplib` to make it asynchronous. If we use the second one, we end up with an additional dependency anyway.

Reply:
Let's use `smtplib` for now and we can come back to this later.